.github/workflows/pr.yml

```
@@ -0,0 +1,27 @@
+name: PR Workflow
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
```

```suggestion
  lint:
```

We dont build anything just yet. We only lint